### PR TITLE
DOC-1930: changelog.adoc

### PR DESCRIPTION
**Pull request by brian-forte**
Ticket: DOC-1930: Tiny 6.4 Community docs release.

Changes:
* added the TinyMCE 6.4-specific changes to `changelog.adoc`.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
~- [ ] `modules/ROOT/nav.adoc` has been updated (if applicable)~
~- [ ] Files has been included where required (if applicable)~
~- [ ] Files removed have been deleted, not just excluded from the build (if applicable)~
~- [ ] (New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -4,6 +4,91 @@
...
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4 - 2023-03-15
+
+=== Added
+* New `tree` component that can be used in dialog body panel.
+* renderUI property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
+* New `isEditable` API to `editor.selection` that returns true or false if the current selection is editable.
+* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button apis.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:14]:
```suggestion
* New `setText` and `setIcon` methods added to menu button and toolbar button API.
```

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -4,6 +4,91 @@
...
+* New `highlight_on_focus` option which enables highlighting the content area on focus.
+* New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.
+* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
+* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* New `group` and `togglebutton` in view.
+* New `togglebutton` in dialog footer buttons.
+* Added `toggleFullscreen` to dialog API.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:21]: Not clear what these new things are

File: modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -4,6 +4,91 @@
...
+
+=== Changed
+* The `link` plugins context menu items no longer appears for links that include elements with a `contenteditable="false"` attribute.
+* The formatting of elements with a `contenteditable="false"` attribute are no longer cloned to new cells when new table rows are created.
+* Changed the color of `@dialog-table-border-color`, and added right padding to the first cell of dialog table.
+* The sidebar element now has the accessibility role `region` when visible and the accessibility role `presentation` when hidden.
+* The `tox-custom-editor` class now has a border highlight when it is selected.
+
+=== Fixed
+* Add support for navigating inside the tree component using arrow keys and shift key.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:43]: This should be moved to `Added` section

File: modules/ROOT/pages/changelog.adoc [hunk 4]
```diff
@@ -4,6 +4,91 @@
...
+* Sometimes the editor would finish initializing before the silver theme would have finished loading.
+* The `searchreplace` modal closed incorrectly when clicking outside of the alert that pops up when no match is found.
+* The text color or background color picker toolbar buttons did not update when the text color or background color was changed using the equivalent commands in the Format menu.
+* The `onSetup` api function would not run when defining custom group toolbar button.
+* An element could be dropped onto the decendants of an element with a `contenteditable="false"` attribute.
+* Checkmark did not show in menu color swatches.
+* The foreground and background menu icons would not properly update to display the last used color.
+* Added new `setIconFill` function to `NestedMenuItemInstanceApi`.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:51]: Move to `Added` section

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc [hunk 5]
```diff
@@ -4,6 +4,91 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4 - 2023-03-15
+
+=== Added
+* New `tree` component that can be used in dialog body panel.
+* renderUI property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
```
[the remark this answers is not in the document]
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: Made the change locally and pushed upstream.

[the remark this answers is not in the document]
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: Made the change locally and pushed upstream.

File: modules/ROOT/pages/changelog.adoc (hunk 1 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: Made the change locally and pushed upstream.

File: modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:21]: They are the same new things as are in changelog.md in the TinyMCE repo. As in [these three lines](https://github.com/tinymce/tinymce/blob/db45bd16f11947b587fede0ec1b9745adc609e30/modules/tinymce/CHANGELOG.md?plain=1#L19)

File: modules/ROOT/pages/changelog.adoc (hunk 3 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:45]: @tiny-ben-tran,

Is it safe to assume you want the equivalent change made to TinyMCE’s `changelog.md`?

File: modules/ROOT/pages/changelog.adoc (hunk 4 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:51]: @tiny-ben-tran,

Is it safe to assume you want the equivalent change made to TinyMCE’s `changelog.md`?

File: modules/ROOT/pages/changelog.adoc (hunk 3 above)
Comment (tiny-ben-tran) [reply modules/ROOT/pages/changelog.adoc:45]: Yes. I left the same comments in two different places. Sorry about that

Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc]: Moved.

File: modules/ROOT/pages/changelog.adoc (hunk 4 above)
Comment (brian-forte) [reply modules/ROOT/pages/changelog.adoc:51]: Moved.

**Review by kemister85 — APPROVED** [review]

**Pull request merged**